compiler/damlc/tests/daml-test-files/NumericLitPolyTooLarge.daml

```diff
@@ -1,15 +0,0 @@
--- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
```

Comment:
What happens for literals that are too large or too small now?

Reply:
Poly literals aren't allowed, this test was only to test those.

Fixed scale literals that are too large or too small are already tested :)